alibi/models/tensorflow/actor_critic.py

```diff
@@ -1,9 +1,15 @@
+from abc import ABC
+
 import tensorflow as tf
 import tensorflow.keras as keras
 
 
-class Actor(keras.layers.Layer):
-    """ Actor network. """
+class Actor(keras.Model, ABC):
```

Comment:
Is the `ABC` here due to `mypy` complaints? Should be a way round it... (same for `Critic`).

Reply:
forgot to delete it. The complaint was from pycharm to implement all methods.